sky/serve/serve_utils.py

```diff
@@ -522,31 +521,25 @@ def cluster_is_up() -> bool:
 def stream_replica_logs(service_name: str,
                         replica_id: int,
                         follow: bool,
-                        skip_local_log_file_check: bool = False) -> str:
+                        skip_replica_log_file_check: bool = False) -> str:
```

Comment:
Do we still need this? Should we always check if the replica log exists?

Reply:
Good catch, seems like we don't need to always check. If we remove, we can 
1. If launch log file exists, print out launch log file and tail logs from replica. 
2. If replica log file exists, print out the replica log file.
3. Don't find any of the aforementioned files and return string indicating that the replica does not exist

If this makes sense, I can make this change and test it

Reply:
I was actually a bit confused as to why the `skip_local_log_file_check` was ever needed as a param since I never saw the `stream_replica_logs` method define this param, so I think it always defaulted to `False`

Reply:
To the second reply - good catch! This is a deprecated param that I forgot to delete in my development 🫥 Previously, I was calling `stream_replica_logs` directly in the `replica_managers::_terminate_replica` function. Now it calls backend functions instead. Please delete it 🫡

Reply:
So for 1) in the first reply, we always run this code right?
https://github.com/skypilot-org/skypilot/blob/e7c49f3034f005578c6c9456c8bc859a31a5002d/sky/serve/serve_utils.py#L588-L599

And after we tail logs from the replica? 
https://github.com/skypilot-org/skypilot/blob/e7c49f3034f005578c6c9456c8bc859a31a5002d/sky/serve/serve_utils.py#L601-L617

Reply:
Yeah. This function will only be called when the replica cluster is found.

Reply:
Updated! PTAL @cblmemo 🫡